pkg/shares/non_interactive_defaults_test.go

```diff
@@ -361,8 +410,72 @@ func TestMinSquareSize(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(fmt.Sprintf("shareCount %d", tc.shareCount), func(t *testing.T) {
-			got := MinSquareSize(tc.shareCount)
+			got := BlobMinSquareSize(tc.shareCount)
 			assert.Equal(t, tc.want, got)
 		})
 	}
 }
+
+func TestSubTreeWidth(t *testing.T) {
+	type testCase struct {
+		shareCount int
+		want       int
+	}
+	testCases := []testCase{
+		{
+			shareCount: 0,
+			want:       1,
+		},
+		{
+			shareCount: 1,
+			want:       1,
+		},
+		{
+			shareCount: 2,
+			want:       1,
+		},
+		{
+			shareCount: appconsts.SubtreeRootThreshold,
+			want:       1,
+		},
+		{
+			shareCount: appconsts.SubtreeRootThreshold + 1,
+			want:       2,
+		},
+		{
+			shareCount: appconsts.SubtreeRootThreshold - 1,
+			want:       1,
+		},
+		{
+			shareCount: appconsts.SubtreeRootThreshold * 2,
+			want:       2,
+		},
+		{
+			shareCount: (appconsts.SubtreeRootThreshold * 2) + 1,
+			want:       4,
+		},
+		{
+			shareCount: (appconsts.SubtreeRootThreshold * 3) - 1,
+			want:       4,
+		},
+		{
+			shareCount: (appconsts.SubtreeRootThreshold * 4),
+			want:       4,
+		},
+		{
+			shareCount: (appconsts.SubtreeRootThreshold * 5),
+			want:       8,
+		},
+		{
+			shareCount: (appconsts.SubtreeRootThreshold * appconsts.DefaultMaxSquareSize) - 1,
+			want:       128,
+		},
+	}
+	for i, tc := range testCases {
+		t.Run(fmt.Sprintf("shareCount %d", tc.shareCount), func(t *testing.T) {
+			got := SubTreeWidth(tc.shareCount)
+			assert.Equal(t, tc.want, got, i)
+			assert.GreaterOrEqual(t, got, appconsts.DefaultMinSquareSize)
```

Comment:
This isn't relevant anymore. Perhaps you want to assert:
```suggestion
			assert.GreaterOrEqual(t, got, BlobMinSquareSize(tc.shareCount))
```

Reply:
https://github.com/celestiaorg/celestia-app/pull/1604/commits/74c4b248a430183cef85ed0de6d06e9c37b2b255

Reply:
good catch!

Reply:
As long as we're covering for it in our test cases then I'm good